tools/offcputime_example.txt

```diff
@@ -11,7 +11,7 @@ Here is some example output. To explain what we are seeing: the very first
 stack trace looks like a page fault (do_page_fault() etc) from the "chmod"
 command, and in total was off-CPU for 13 microseconds.
```

Comment:
Add to the paragraph. "The -K option was used to only match kernel stacks". (I try to make sure the 1st example explains everything; later examples don't have to).